content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,376 @@
+---
+title: "dbt models for CHT Applications"
+linkTitle: "dbt Models"
+weight: 2
+description: >
+  Guide for building dbt models for CHT applications
+relatedContent: >
+  core/overview/db-schema
+  apps/reference/app-settings/hierarchy
+---
+
+## Overview
+
+[CHT Sync]({{< relref "core/overview/cht-sync" >}}) copies data from CouchDB to a relational database. It initially stores the document data from CouchDB in a `jsonb` column in a single table. This is not possible to query for analytics, so it uses [dbt](https://www.getdbt.com/) to convert the document data to a relational database format.
+
+[CHT Pipeline](https://github.com/medic/cht-pipeline) defines a dbt project, which contains model files for the data schema described [in the Database schema conventions]({{< ref "core/overview/db-schema" >}}).
+Forms may be specific to each CHT application; additional models will need to be developed to analyze data from responses to these custom forms.
+One additional model will be needed for each form, and for any aggregations, dashboards, or reusable views that use those form responses as input.
+If using the [configurable contact hierarchy]({{< ref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}), it may also be useful to add models for other contact types.
+
+## Setup
+
+To create application specific models, create a  [new dbt project](https://docs.getdbt.com/reference/commands/init).   Edit the `packages.yml` in your new dbt project to add the [CHT Pipeline](https://docs.getdbt.com/docs/build/packages)  as a dependency.   So that you can track changes in your models, put your dbt new project in a GitHub repository (it may be public or private).
```

Comment:
```suggestion
To create application specific models, create a  [new dbt project](https://docs.getdbt.com/reference/commands/init). Edit the `packages.yml` in your new dbt project to add [cht-pipeline](https://docs.getdbt.com/docs/build/packages) as a dependency. So that you can track changes in your models, put your dbt new project in a GitHub repository (it may be public or private).
```